### PR TITLE
reject services which have a status == not-found

### PR DESCRIPTION
**Pull request by richm**

### CODE REVIEW

**Comment by richm** [comment]
candidate fix for https://github.com/linux-system-roles/timesync/issues/128

**Comment by richm** [comment]
The ansible-2.12 failures are fixed by https://github.com/linux-system-roles/timesync/pull/130 - the other failures are known

**Review by nhosoi — APPROVED** [review]
lgtm

File: tasks/main.yml [hunk 1]
```diff
@@ -20,7 +20,9 @@
 - name: Set variable `timesync_services` with filtered uniq service names
   set_fact:
     timesync_services: "{{ ansible_facts.services.values() |
-          selectattr('name', 'defined') | map(attribute='name') |
+          selectattr('name', 'defined') |
+          rejectattr('status', 'match', '^not-found$') |
```
Comment (spetrosi) [inline tasks/main.yml:24]: Looking at the fix in the service_fact module upstream, https://github.com/ansible/ansible/blob/stable-2.12/lib/ansible/modules/service_facts.py#L212, I think the following must be added also:
*  `rejectattr('status', 'match', '^masked$')`
* `rejectattr('status', 'match', '^failed$')`
Same workaround is required here:
https://github.com/linux-system-roles/storage/blob/master/tasks/main-blivet.yml#L54
I've opened AAP-1128 on Red Hat's Jira to cherry-pick this to ealrier Ansibles.

**Pull request merged**